app.py

```diff
@@ -508,7 +508,7 @@ def error(self, code, message, **kwargs):
                 description: Upload field not found
         """
         mnm.uploads_total.inc()
-        self.request_id = self.request.headers.get('x-rh-insights-request-id', uuid.uuid4().hex)
+        self.request_id = self.request.headers.get('x-rh-insights-request-id', "00000" + uuid.uuid4().hex)
```

Comment:
I don't see anything technically wrong with this.  However, I do have a couple of suggestions:

1)  You might consider making the prefix a "constant" ...something like "GENERATED_REQUEST_ID_PREFIX".  You could then use this name in the code instead of the hardcoded string.  I use constants/names because I have a tendency to "booger" up the strings by misspelling the words, missing a "0", etc.

2) You might consider moving the request_id generation into a method:
  def generate_request_id():
    return GENERATED_REQUEST_ID_PREFIX + uuid.uuid4().hex

3) You might consider moving the retrieval of the x-rh-insights-request-id to a method:
  def get_request_id(request):
    return request.headers.get("x-rh-insights-request-id", generate_request_id())

I try to use method names (instead of comments) to try to end up with self-documenting code (that's the theory at least  :)  ).  The other nice thing that comes out of splitting the logic into more (smaller) methods is containment and with containment...you end up with code that is easier to move around (when you figure out it doesn't belong where its currently located) and code that is easier to unit test.